patches/minecraft/net/minecraft/entity/player/PlayerEntity.java.patch

```diff
@@ -210,6 +210,16 @@
                 }
  
                 f = f + f1;
+@@ -1081,7 +_,8 @@
+                      float f3 = 1.0F + EnchantmentHelper.func_191527_a(this) * f;
+ 
+                      for(LivingEntity livingentity : this.field_70170_p.func_217357_a(LivingEntity.class, p_71059_1_.func_174813_aQ().func_72314_b(1.0D, 0.25D, 1.0D))) {
+-                        if (livingentity != this && livingentity != p_71059_1_ && !this.func_184191_r(livingentity) && (!(livingentity instanceof ArmorStandEntity) || !((ArmorStandEntity)livingentity).func_181026_s()) && this.func_70068_e(livingentity) < 9.0D) {
++                        double attackReachSq = this.func_110148_a(net.minecraftforge.common.ForgeMod.ATTACK_REACH.get()).func_111126_e() * this.func_110148_a(net.minecraftforge.common.ForgeMod.ATTACK_REACH.get()).func_111126_e();
++                        if (livingentity != this && livingentity != p_71059_1_ && !this.func_184191_r(livingentity) && (!(livingentity instanceof ArmorStandEntity) || !((ArmorStandEntity)livingentity).func_181026_s()) && this.func_70068_e(livingentity) < attackReachSq) {
```

Comment:
This can be combined into one line, a long one yes but it reduces the number of lines to add.

Reply:
Okay, I was just under the impression that I should've avoided making a really long line like that

Reply:
I don't believe that's the case, but I'm not 100% sure on my statement. I'm just remembering previous PRs.

Reply:
This is a gigantic mess! Split it up for readability, jesus.

Reply:
Done.